packages/next-admin/src/utils/prisma.ts

```diff
@@ -289,12 +299,14 @@ const preparePrismaListRequest = <M extends ModelName>(
   let where = {};
   const list = options?.model?.[resource]?.list as ListOptions<M>;
   select = selectPayloadForModel(resource, list, "object");
-  where = createWherePredicate({
-    resource,
-    options,
-    search,
-    otherFilters: fieldFilters,
-  });
+  where =
+    getWherePredicateFromQueryParams(searchParams) ||
```

Comment:
Maybe it's better to combine 'classic' filters as search and advanced ones, instead of excluding 'classic' filters if there are advanced ones